README.md

```diff
@@ -34,7 +34,7 @@ If your app will not function without nfc capabilities, set `android:required` a
 ## Usage
 ```js
 import PassportReader from 'react-native-passport-reader'
-// { scan, cancel, isSupported }
+// { scan, cancel, isSupported, constants }
```

Comment:
while we're here, i think `isSupported` is part of `constants` rather than on its own
```suggestion
// { scan, cancel, constants }
```